src/history/test/HistoryTests.cpp

```diff
@@ -1129,6 +1193,10 @@ TEST_CASE("Publish catchup via s3", "[!hide][s3]")
 TEST_CASE("HAS in publishqueue remains in pristine state until publish",
           "[history]")
 {
+    if (!isTestingLatest())
```

Comment:
I don't think this is right, see `for_versions_with_differing_bucket_logic` on line 1217.

Reply:
Yeah good catch. Removed.